app/server/endpoint/src/main/java/io/syndesis/server/endpoint/v1/handler/continuousdelivery/ContinuousDeliveryHandler.java

```diff
@@ -0,0 +1,184 @@
+
+    /**
+     * Delete an environment tag associated with this integration.
+     */
+    @DELETE
+    @Path("releasetags/{id}/{env}")
+    void deleteReleaseTag(@NotNull @PathParam("id") @ApiParam(required = true) String integrationId, @NotNull @PathParam("env") @ApiParam(required = true) String environment);
+
+    /**
+     * Tag an integration for release to target environments.
+     */
+    @POST
+    @Path("releasetags/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    Map<String, ContinuousDeliveryEnvironment> tagForRelease(@NotNull @PathParam("id") @ApiParam(required = true) String integrationId,
+                                                @NotNull @ApiParam(required = true) List<String> environments);
+
+    /**
+     * Export integrations to a target environment.
+     */
+    @GET
+    @Path("export/{env}/export.zip")
+    @Produces(MediaType.APPLICATION_OCTET_STREAM)
+    StreamingOutput exportResources(@NotNull @PathParam("environment") @ApiParam(required = true) String environment,
+                           @QueryParam("all") @ApiParam() boolean exportAll) throws IOException;
+
+    /**
+     * Import integrations into a target environment.
+     */
+    @POST
+    @Path("import")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    ContinuousDeliveryImportResults importResources(@Context SecurityContext sec, @NotNull @ApiParam ImportFormDataInput formInput) throws IOException;
+
+    /**
+     * Configure a connection.
+     */
+    @POST
+    @Path("connections/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    ConnectionOverview configureConnection(@Context SecurityContext sec, @NotNull @PathParam("id") @ApiParam String connectionId, @NotNull @ApiParam Map<String, String> properties) throws IOException;
+
+    /**
+     * Get Integration state.
+     */
+    @GET
+    @Path("integrations/{id}/state")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    IntegrationDeploymentStateDetails getIntegrationState(@Context SecurityContext sec, @NotNull @PathParam("id") @ApiParam String integrationId) throws IOException;
+
+    /**
+     * Start Integration.
+     */
+    @PUT
+    @Path("integrations/{id}/deployments")
+    @Produces(MediaType.APPLICATION_JSON)
+    IntegrationDeployment publishIntegration(@Context final SecurityContext sec, @NotNull @PathParam("id") @ApiParam(required = true) final String integrationId);
+
+    /**
+     * Stop Integration.
+     */
+    @POST
+    @Path("integrations/{id}/deployments/{version}")
+    @Produces(MediaType.APPLICATION_JSON)
+    void stopIntegration(@Context final SecurityContext sec, @NotNull @PathParam("id") @ApiParam(required = true) final String integrationId,
+                                          @NotNull @PathParam("version") @ApiParam(required = true) final int version);
```

Comment:
Is `version` needed?

Reply:
Yeah, it can figure it out on it's own, so removed it